core/types/system_tx.go

```diff
@@ -0,0 +1,104 @@
+package types
+
+import (
+	"bytes"
+	"math/big"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/rlp"
+)
+
+type SystemTx struct {
+	ChainID *big.Int
```

Comment:
If we change to "sequencer produces tx internally" approach, then we can drop `chainID` and the signature.

Reply:
The "sequencer produces tx internally" approach is not at odds with having multiple sequencers.

Reply:
agreed, this is the option taken in latest update.